index.html

```diff
@@ -50,7 +47,263 @@ <h6>LET'S CONNECT</h6>
+
+        <div class="project-title">
+         <h2>Multi-Post <br> Stories</h2>
+        </div>
+
+         <section class="small-title">
+             <p>CANOPY</p>
+             <div class="list">
+                 <ul>
+                 <li>Back End Dev</li>
+                 <li>2015</li>
+                 </ul>
+                </div>
+         </section>
+
+
+        <div class="describe-project"> 
+           <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+        </div>
+ 
+        <div class="small-buttons">
+           <ul>
+              <li> html</li> 
+               <li>css</li> 
+               <li>javascript</li>
+            </ul> 
+        </div>
+ 
+        <div class="big-button">
+                <button>
+                    See Project
+                </button>
+            </div>
+ 
+     </section>
+
+     <section class="works-tonic">
+       
+        <figure class="image">
+            <img src="imageone.png" alt="imageone">
+        </figure>
+        
+        <section class="project-title">
+         <h2>Tonic</h2>
+         <div class="small-title">
+             <p>CANOPY</p>
+             <div class="list">
+                 <ul>
+                 <li>Back End Dev</li>
+                 <li>2015</li>
+                 </ul>
+             </div>
+                 
+         </div>
+ 
+        <div class="describe-project"> 
+           <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+         </div>
+ 
+        <div class="small-buttons">
+         <ul>
+             <li> html</li> 
+              <li>css</li> 
+              <li>javascript</li>
+           </ul> 
+         </div>
  
+        <div class="big-button">
+                <button>
+                    See Project
+                </button>
+             </div>
+ 
+     </section>
+
+     <section class="works-stories">
+       
+        <figure class="image">
+            <img src="imagefour.png" alt="imageone">
+        </figure>
+        
+        <section class="project-title">
+         <h2>Multi-Post <br> Stories</h2>
+         <div class="small-title">
+             <p>CANOPY</p>
+             <div class="list">
+                 <ul>
+                 <li>Back End Dev</li>
+                 <li>2015</li>
+                 </ul>
+                </div>
+            </div>
+ 
+        <div class="describe-project"> 
+           <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+        </div>
+ 
+        <div class="small-buttons">
+            <ul>
+                <li> html</li> 
+                 <li>css</li> 
+                 <li>javascript</li>
+              </ul> 
+            </div>
+ 
+        <div class="big-button">
+                <button>
+                    See Project
+                </button>
```

Comment:
On the buttons, kindly implement the hover `color` and make it a `cursor: pointer`.